templates/oe-corporate-blocks-eu-footer.html.twig

```diff
@@ -0,0 +1,58 @@
+{#
+/**
+ * @file
+ * Default theme implementation for rendering corporate blocks footer.
+ *
+ * Available variables:
+ * - corporate_footer: Array with structure compatible to ECL ec-components-footer component.
+ *   - contact: EU contact data for corporate footer.
+ *   - social_media: Social media links for corporate footer.
+ *   - institution_links: Data about EU for corporate footer.
+ *   - other_links: Data about EU for corporate footer.
+ *
+ * @see template_preprocess_corporate_blocks_eu_footer()
+ *
+ * @ingroup themeable
+ */
+#}
+
+<div class="footer__corporate-top--eu">
+  {% if corporate_footer.contact %}
+    <ul class="footer__list">
+      {% for menu_item in corporate_footer.contact %}
+        <li class="footer__list-item">
+          {{ menu_item|raw }}
```

Comment:
Why we need the |raw here?

Reply:
because `menu_item` contains a pure HTML (not a renderable array).

Reply:
This variable will not be escaped.

Reply:
corrected